ath.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euxo pipefail
```

Comment:
```suggestion
set -o errexit
set -o nounset
set -o pipefail
set -o xtrace
```

Same argument as for long options, easier to understand if you're unfamiliar with the flags.